CMakeLists.txt

```diff
@@ -55,16 +55,24 @@ set(EXE_SRCS
     macros/runDummy.C
     macros/crappyTest.C
     macros/convertToCTSEvents.C
+    macros/calibrateCTSEvents.C
     macros/readCTSEventTree.C
     macros/tryAroundTracker.C
+    macros/plotCTSEvent.C
+    macros/convertToClusterEvents.C
+    macros/plotClusterEvent.C
     )
 
 set(EXE_NAMES
     runDummy
     runCrappyTest
     convertToCTSEvents
+    calibrateCTSEvents
     readCTSEventTree
     tryAroundTracker
+    plotCTSEvent
+    convertToClusterEvents
+    plotClusterEvent
```

Comment:
Would be nice to have the `convert`..., `plot`... etc macros together in the lists. 
e.g. 

- `convertToCTSEvents`
- `convertToClusterEvents`
- .
- .
- .
- `plotCTSEvents`
- `plotClusterEvent`